app/src/main/java/com/example/movieapp/ListAdapter.kt

```diff
@@ -0,0 +1,52 @@
+package com.example.movieapp
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import android.widget.ImageButton
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.example.movieapp.crawler.pojo.Movie
+import com.squareup.picasso.Picasso
+import kotlinx.android.synthetic.main.list_item.view.*
+
+class ListAdapter(private val list: List<Movie>) : RecyclerView.Adapter<MovieViewHolder>() {
+
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): MovieViewHolder {
+        val inflater = LayoutInflater.from(parent.context)
+        return MovieViewHolder(inflater, parent)
+    }
+
+    override fun onBindViewHolder(holder: MovieViewHolder, position: Int) {
+        val movie: Movie = list[position]
+        holder.bind(movie)
+    }
+
+    override fun getItemCount(): Int = list.size
+
+}
+
+class MovieViewHolder(inflater: LayoutInflater, parent: ViewGroup) :
+    RecyclerView.ViewHolder(inflater.inflate(R.layout.list_item, parent, false)) {
+    private val TAG = "ListAdapter"
+
+    private var mTitleView: TextView? = null
```

Comment:
I know it's in a lot of tutorials but the Hungarian notation( putting 'm' at the start of the var name) is horrible and you shouldn't use it

Reply:
done, but what do you prefer ?

Reply:
done